relevanceai/dataset_api/dataset_write.py

```diff
@@ -544,3 +569,26 @@ def delete(self):
 
         """
         return self.datasets.delete(self.dataset_id)
+
+    def insert_df(self, dataframe, *args, **kwargs):
```

Comment:
Just do this if you really want it. 

```
insert_df = insert_pandas_dataframe
```

However, I don't think this is good because:

```
df.insert_df
```
Looks like a really weird function - like you're inserting another `dataset` object which we dont suport. I personally think this should not be included because `df` can also be a nother of things (dask dataframe, Vaex data stuff, etc. etc)

Reply:
I have done this `insert_df = insert_pandas_dataframe` since customer success requested it